src/js/engine/engine.js

```diff
@@ -54,6 +54,17 @@ const sendFileOpened = async (
   sender.send(ipcChannel, action);
 };
 
+// Invisible character U+2800 being used in line.replace
+const replaceEmptyLinesWithHiddenChar = arr => {
+  const regexList = [/^\s*$/];
+  return arr.map(line => {
+    const isMatch = regexList.some(rx => {
+      rx.test(line);
```

Comment:
I tried it with both rows containing "" and " ". And it works in my logviewer.
It did not work at first though, but after adding a return on row 62 no errors occured. 
`return rx.test(line)`